.travis.yml

```diff
@@ -17,12 +17,7 @@ env:
     #- TARGET_PLATFORM=ol/8
     - TARGET_PLATFORM=el/7
     - TARGET_PLATFORM=ol/7
-    - TARGET_PLATFORM=el/6
-    # Oracle Linux 6 is old and it doesn't work with static assert:
-    # https://github.com/citusdata/pg_auto_failover/issues/200
-    # - TARGET_PLATFORM=ol/6
```

Comment:
Can we bring those commented lines back ?

Reply:
Oh, my bad. I will keep ol6 related comments there